Site_Code/script.js

```diff
@@ -0,0 +1,466 @@
+createProjectMainCard("Grid-Tab-2", "Cyber Discovery Bot", "The bot for the Cyber Discovery Community Discord Server. It has a variety of important and fun features. For example, it can get the briefing for a CyberStart Assess Level, or fetch an XKCD. Relax, take a load off and join our discord. Invite link in card menu.", "cdbotmain_card_image", "https://github.com/CyberDiscovery/cyberdisc-bot", "https://github.com/login?return_to=%2FCyberDiscovery%2Fcyberdisc-bot", "https://github.com/CyberDiscovery/cyberdisc-bot/issues", "Server Invite", "http://discord.gg/Kf8n5rT", "exit_to_app");
+createImageCardMain("Grid-Tab-2", "discord_bot_assess_card_image", "Discuss assess challenges!");
+createProjectTextFieldCards("Grid-Tab-2", "Search HIBP", "Our bot can check if you have an account that has been compromised in a data breach, courtesy of Microsoft MVP Troy Hunt's HaveIBeenPwned service. You can try it out here!", "hibp_search_card_image", "Email", checkPwnState);
+createProjectTextFieldCards("Grid-Tab-2", "Fetch XKCD", "Our bot can can fetch an XKCD by number or a random one. You can try out the same functionality here, leaving the number-field blank for a random one.", "xkcd_card_fetch_image", "Number", fetchXkcd);
+createProjectMainCard("Grid-Tab-2", "Maths Bot", "A Discord Maths Bot written in Python. designed to give problems from the Kings Maths School Seven Day Maths website. This includes the current weekly challenge, as well as a random problem from their archive.", "cdmathsbot_card_image", "https://github.com/CyberDiscovery/Discord-Maths-Bot", "https://github.com/login?return_to=%2FCyberDiscovery%2FDiscord-Maths-Bot", "https://github.com/CyberDiscovery/Discord-Maths-Bot/issues", "Math Challenge", "https://www.kcl.ac.uk/mathsschool/weekly-maths-challenge/weekly-maths-challenge.aspx", "question_answer");
+createImageCardMain("Grid-Tab-2", "discord_dwouca_owo_gang_lmao", "Weekly maths challenge!");
+createLinkCard("Grid-Tab-2", "API Documentation", "Are you good at Python? Want to program your own bot? Want to help us improve ours, but don't know much of Discord's API? In that case, the developer documentation is a great place to start!", "discord_api_docs_card_image", "https://discordapp.com/developers/docs/intro");
+createGitHubContribCard("Grid-Tab-2", "cyberdisc-bot");
+createProjectMainCard("Grid-Tab-3", "Android & IOS App", "Never miss a deadline again with the unofficial Cyber Discovery App for Android and iOS. You can build it from source, get from the play store or sideload a prebuilt release.", "cdapp_main_card_image", "https://github.com/CyberDiscovery/Cyber-Discovery-App", "https://github.com/login?return_to=%2FCyberDiscovery%2FCyber-Discovery-App", "https://github.com/CyberDiscovery/Cyber-Discovery-App/issues", "Latest Release", "https://play.google.com/store/apps/details?id=com.danielmilnes.cyberdiscovery", "arrow_downward");
```

Comment:
```suggestion
createProjectMainCard("Grid-Tab-3", "Android & IOS App", "Never miss a deadline again with the unofficial Cyber Discovery App for Android and iOS. You can build it from source, get from the Play Store or sideload a prebuilt release.", "cdapp_main_card_image", "https://github.com/CyberDiscovery/Cyber-Discovery-App", "https://github.com/login?return_to=%2FCyberDiscovery%2FCyber-Discovery-App", "https://github.com/CyberDiscovery/Cyber-Discovery-App/issues", "Latest Release", "https://play.google.com/store/apps/details?id=com.danielmilnes.cyberdiscovery", "arrow_downward");
```
Is it possible to also link to the Google Play here? I don't know if MDL can do that.

Reply:
Sorry, what is the change needed here?